src/Commands/CommandFactory.cs

```diff
@@ -66,17 +66,37 @@ public CommandFactory(IServiceProvider serviceProvider, ILogger<CommandFactory>
 
     public IReadOnlyDictionary<string, IBaseCommand> AllCommands => _commandMap;
 
-    public IReadOnlyDictionary<string, IBaseCommand> GroupCommands(string groupName)
+    public IReadOnlyDictionary<string, IBaseCommand> GroupCommands(string commaSeparatedGroupNames)
     {
-        foreach (CommandGroup group in _rootGroup.SubGroup)
+        var groupNames = commaSeparatedGroupNames.Split(',', StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
```

Comment:
Should we be doing the string split here? IIRC, we're using CommandLine options class, which should support this case.

Reply:
https://learn.microsoft.com/en-us/dotnet/standard/commandline/syntax#multiple-arguments

Reply:
Agree we should look into this if supported.  Otherwise string split should be `ok` for the short term till this type of arg is supported.

Reply:
I can look into implement the multi-service support using the multiple arguments syntax but it will take more time. Are you ok with shipping this as a temporary solution and switch to the new syntax in a follow up PR?